modules/@apostrophecms/page/index.js

```diff
@@ -976,6 +964,93 @@ database.`);
           }
         }
       },
+      // A method to return a target page object based on a passed `targetId`
+      // value. `position` is used to prevent attempts to move after the trash
+      // "page."
```

Comment:
... Which works even for numeric positions because this is called again with a non-numeric position after getTargetAndPosition succeeds. OK.